modules/color-me/configs/strings.json

```diff
@@ -26,6 +26,23 @@
       "type": "string",
       "allowEmbed": true
     },
+    {
+      "name": "createdNoIcon",
+      "humanName": {
+        "en": "Role created without icon",
+        "de": "Rolle ohne Icon erstellt"
+      },
+      "default": {
+        "en": "Your role was created successfully, but your role icon was not used, as this requires the guild to be boost level 2 or higher.",
+        "de": "Deine Rolle wurde erfolgreich erstellt, aber dein Rollen-Icon wurde nicht verwendet, da hierfür der Server mindestens Boostlevel 2 besitzen muss."
+      },
+      "description": {
+        "en": "This messages gets send when a booster sucessfully created their custom role, but the guild has not enough boosts to use role icons",
```

Comment:
```suggestion
        "en": "This messages gets send when a booster successfully created their custom role, but the guild has not enough boosts to use role icons",
```

Reply:
I don't like writing good text for users :hAA: